### PR TITLE
Redenominate to 10 decimals

### PR DESCRIPTION
**Pull request by F3Joule**
### Release Checklist

- [x] Verify `spec_version` has been incremented since the last release.
- [x] Verify that `Cargo.toml` packages version increased.

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -259,7 +259,7 @@ impl pallet_timestamp::Config for Runtime {
 }
 
 parameter_types! {
-	pub const ExistentialDeposit: u128 = 10 * CENTS;
+	pub const ExistentialDeposit: u128 = 100 * CENTS;
```
Comment (TarekkMA) [inline runtime/src/lib.rs:262]: Why increase value?

File: runtime/src/constants.rs [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-	pub const CENTS: Balance = DOLLARS / 100;      // 1_000_000_000
-	pub const MILLICENTS: Balance = CENTS / 1_000; // 1_000_000
+	pub const UNITS: Balance = 10_000_000_000;
+	pub const DOLLARS: Balance = UNITS;            // 10_000_000_000
+	pub const CENTS: Balance = DOLLARS / 100;      // 100_000_000
+	pub const MILLICENTS: Balance = CENTS / 1_000; // 100_000
 
 	pub const fn deposit(items: u32, bytes: u32) -> Balance {
-		items as Balance * 15 * CENTS + (bytes as Balance) * 6 * CENTS
+		items as Balance * 20 * DOLLARS + (bytes as Balance) * 100 * MILLICENTS
```
Comment (TarekkMA) [inline runtime/src/constants.rs:10]: why this change?

**Review by TarekkMA — CHANGES_REQUESTED** [review]

File: runtime/src/constants.rs (hunk 2 above)
Comment (F3Joule) [reply runtime/src/constants.rs:10]: We don't use `deposit` function for now, but I aligned this calculations with https://github.com/paritytech/polkadot/blob/16e6ce0ca0330aad1e0beea2b064f704ce92f207/runtime/polkadot/constants/src/lib.rs#L36

File: runtime/src/lib.rs (hunk 1 above)
Comment (F3Joule) [inline runtime/src/lib.rs:262]:
```suggestion
	pub const ExistentialDeposit: u128 = 10 * CENTS;
```

File: runtime/src/constants.rs (hunk 2 above)
Comment (TarekkMA) [reply runtime/src/constants.rs:10]: Okay then, good job ❤️

**Review by TarekkMA — APPROVED** [review]

**Review by siman — APPROVED** [review]
